docker-compose.yaml

```diff
@@ -0,0 +1,111 @@
+version: '3'
+services:
+  observatorium:
+    build: .
+    command:
+      - --metrics.query.endpoint=http://thanos-querier:9090
+      - --metrics.write.endpoint=http://thanos-receive:19291
+      - --log.level=debug
+    ports:
+      - 8080:8080
+    links:
+      - thanos-querier
+      - thanos-receiver
+
+  prometheus:
+    image: prom/prometheus:v2.14.0
+    command:
+      - --config.file=/config/prometheus.yml
+      - --log.level=debug
+    volumes:
+      - ./data/test/config:/config
+    ports:
+      - 9090
+    links:
+      - observatorium
+      - thanos-receiver
```

Comment:
we need links to thanos-store and thanos-querier too, since they are scraped for metrics